### ISSUE TITLE
sample queries and history items are inaccessible via keyboard

### ISSUE THREAD

**Issue by ElinorW** [microsoftgraph/microsoft-graph-explorer-v4#3648]
- The sample query items have no focus when using keyboard to navigate.
- The history items have no keyboard navigation access
